### PR TITLE
ffmpeg-qsv/gst-msdk: enable max frame size

### PR DESCRIPTION
**Pull request by yefengxx**
lib/parameters: added functions for max_frame_size
ffmpeg-qsv: added max_frame_size for avc/hevc enc
gst-msdk: added max_frame_size for avc/hevc enc

Signed-off-by: Xu Yefeng <yefengx.xu@intel.com>

### CODE REVIEW

**Comment by yefengxx** [comment]
@uartie @FocusLuo can you review this for me?

File: lib/gstreamer/msdk/encoder.py [hunk 1]
```diff
@@ -63,12 +63,13 @@ def maxrate(self):
   ladepth = property(lambda s: s.ifprop("ladepth", " rc-lookahead={ladepth}"))
   tilecols = property(lambda s: s.ifprop("tilecols", " num-tile-cols={tilecols}"))
   tilerows = property(lambda s: s.ifprop("tilerows", " num-tile-rows={tilerows}"))
+  maxFrameSize = property(lambda s: s.ifprop("maxFrameSize", " max-frame-size={maxFrameSize}"))
```
Comment (uartie) [inline lib/gstreamer/msdk/encoder.py:66]: Camel case name doesn't match general style.  How about `max_frame_size` or `maxfrsize` or `maxframesize`?

**Review by uartie — CHANGES_REQUESTED** [review]

File: lib/gstreamer/msdk/encoder.py (hunk 1 above)
Comment (yefengxx) [reply lib/gstreamer/msdk/encoder.py:66]: > Camel case name doesn't match general style. How about `max_frame_size` or `maxfrsize` or `maxframesize`?

@uartie thanks for your advice. I will modify later.

**Comment by yefengxx** [comment]
new pr: https://github.com/intel/vaapi-fits/pull/409

**Pull request closed without merging**